src/System.Diagnostics.DiagnosticSource/src/System/Diagnostics/DiagnosticListener.cs

```diff
@@ -34,6 +34,16 @@ namespace System.Diagnostics
     /// </summary>
     public class DiagnosticListener : DiagnosticSource, IObservable<KeyValuePair<string, object>>, IDisposable
     {
+#if false
+        /// <summary>
+        /// This is the DiagnosticListener that is used by default by the class library.   
+        /// Generally you don't want to make your own but rather have everyone use this one, which
+        /// ensures that everyone who wished to subscribe gets the callbacks.  
+        /// The main reason not to us this one is that you WANT isolation from other 
+        /// events in the system (e.g. multi-tenancy).  
+        /// </summary>
+        public static DiagnosticListener DefaultListener { get { return s_default; } }
+#endif 
```

Comment:
Same question about these ```#if false``` blocks... they were just removed in https://github.com/dotnet/corefx/commit/8a39134f5b6058ba4ec1b01153ee78597b9ba295... why are they being added back?